web_theme_social/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+{
+'name': 'Odoo.Social Theme',
+'author': 'Rossa S.A.',
+'category': 'Hidden',
+'version': '8.0.1.0',
+'description':
```

Comment:
remove description and create aproper README file following: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst